.github/workflows/release-embedding-sdk.yml

```diff
@@ -4,32 +4,87 @@ on:
   workflow_dispatch:
     inputs:
       # TODO: Add a version validation, so the workflow won't fail when publishing to npm
-      sdk_version:
-        description: "SDK version (e.g. 0.1.3)"
-        type: string
-        required: true
-      git_ref:
-        description: "A git commit hash that will be used to build the SDK"
-        type: string
+      branch:
+        description: "Branch we want to release the SDK from"
+        type: choice
```

Comment:
Choice would look something like this
<img src="https://private-user-images.githubusercontent.com/1937582/377377792-cbf6a18a-fd72-4258-bcfa-7a87a8cc3407.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkxODAxOTMsIm5iZiI6MTcyOTE3OTg5MywicGF0aCI6Ii8xOTM3NTgyLzM3NzM3Nzc5Mi1jYmY2YTE4YS1mZDcyLTQyNTgtYmNmYS03YTg3YThjYzM0MDcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTdUMTU0NDUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTgyNWJjNDUzNTljMjhlYWJkN2I2YjJhZjA4N2ZkNTkxZmVhMmM2ZDMxNmRmYzQxZmE2ZDM0Y2IwYWI3YzBjOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.eu8NSE4kvFEuHxJ_dNjNtc55uptvH3MYq8PV_Hvwux8" />

Reply:
@WiNloSt fyi, the attached images seems to not be loading. (but I can imagine how that looks like)

Reply:
![image](https://github.com/user-attachments/assets/ea65ccc6-3a83-476e-b81b-3fba3ad7836d)
🤔

Reply:
I found github's hosted images busted sometimes.

Reply:
that's pretty weird - i can see your reply screenshot though so it's good now